enterprise/models.py

```diff
@@ -94,21 +109,33 @@ class EnterpriseCustomerUser(TimeStampedModel):
     """
 
     enterprise_customer = models.ForeignKey(EnterpriseCustomer, blank=False, null=False)
-    user_id = models.PositiveIntegerField(null=False, blank=False)
```

Comment:
The only real-world implementation that comes to mind is the XBlock API/runtime, although the concept of a "user IDA" has been coming up frequently in recent architectural discussions.